docker-in-docker.md

```diff
@@ -0,0 +1,18 @@
+# Docker in Docker
+
+Running Docker in Docker is not something that realy happens that much but is a scenario that can occur. For example when you are using Docker in you Continious Integration tooling. There are some security risk accompanied by this solution which most users are not aware of. In this excersise we are going to take a closer look in the security implications of a set up like this.
+
+![yo dawg](img/docker-meme.jpg)
+
+## Kill other containers!
+
+kill other containers?
+
+## Start other containers!
+
+start other containers?
+
+## Sudo with no audit logging
```

Comment:
Hint: what would happen if you were to, oh, I don't know, start a container with host `/` volume mounted read-write...?

Also: what about using `docker cp` for backdooring the host OS, or for exfiltrating sensitive host OS files? Or move laterally: attack a DB in another container by dumping its contents?